### DISCUSSION TITLE
will this work with the creality touchscreen with no encoder?

### DISCUSSION THREAD

**Discussion by jmartens2** [Jyers/Marlin#460] [Q&A] [answered]
as the title sez i purchased a ender 3 pro custom touchscreen offered at the official creality store and im have trouble getting some of the advanced features back. this is a dwin based board.
please forgive me if im in wrong place to ask this info but as you probably guessed by the lack of proper info im a noobie. thanks for your time if youve read this!

**Comment by Jyers** [comment, marked as answer]
There is currently no support for the new creality touch screen. If at some point I am able to get such a screen it could be possible to port over the code, but at the moment there is not much I can do.

**Reply by bakaneko59** [reply]
Is this question and answer regarding the new screen shown here? https://www.creality3dofficial.com/collections/spare-parts-acc/products/upgraded-touch-panel-screen-display-kits-for-ender-3

**Reply by jmartens2** [reply]
Yup

**Comment by jmartens2** [comment]
Hi thanks for the reply. Where do you live?

**Reply by jmartens2** [reply]
Are you in North America?

**Reply by Jyers** [reply]
Yes, I'm in the US.

**Comment by jmartens2** [comment]
You have my email. I would like to make an offer.

**Reply by Jyers** [reply]
Sure, you can email me at jyers.marlin@gmail.com

**Comment by smooreace** [comment]
Interesting! I doubt that most of the benefits of Jyers firmware would do
anything, much less work, with that screen though. If would be kinda nice
if they said if that was a 4.2.2 board or a 4.2.7

**Reply by jmartens2** [reply]
The picture shows it as 4.2.7 board

**Comment by smooreace** [comment]
Poof... Question asked

**Comment by smooreace** [comment]
It does, but unless it's specifically documented then it can easily be
excused away by a random disclaimer that most sites selling products have
in place.

**Reply by jmartens2** [reply]
Very true

**Reply by jmartens2** [reply]
tried the other marlin/ender suggestion jyers made to me but i cant get my head straight on how to do. i really want to thank jyers for at least being honest and helpful. ill be expanding my search.
